src/Client.js

```diff
@@ -179,7 +183,9 @@ function Client(options) {
     fetch: undefined,
     queryTimeout: null,
     http2SessionIdleTime: http2SessionIdleTime.value,
+    notify_about_new_version: true,
   })
+  notifyAboutNewVersion(options.notify_about_new_version)
```

Comment:
We might need to use camel style for this variable, so it looks the same as other variables?